src/grammar-registry.coffee

```diff
@@ -54,9 +53,9 @@ class GrammarRegistry extends FirstMate.GrammarRegistry
 
   getGrammarPathScore: (grammar, filePath) ->
     return -1 unless filePath
-    filePath = filePath.replace(/\\/g, '/') if process.platform is 'win32'
 
-    pathComponents = filePath.toLowerCase().split(PathSplitRegex)
+    pathComponents = filePath.toLowerCase().split(path.sep)
```

Comment:
Yeah, I'm not sure if `filePath` is always a file path, or if it's sometimes a URL. It might be better to just stick w/ the current approach of converting all separators to `/`.